<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>About me</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>YIQUAN WANG</strong></a>
									<ul class="icons">
										<li><a href="https://twitter.com/Yiquan_W" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="facebook.com/yiquanW/" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="https://github.com/Wangyiquan95" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
										<li><a href="https://www.instagram.com/yiquan1995/" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="https://www.linkedin.com/in/yiquan-w" class="icon brands fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
									</ul>
								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1>About me</h1>
									</header>
									<p><span class="image left"><img src="images/Yiquan.jpg" alt="" /></span>  </p>
									<h2>Professional summary</h2>
									<p>Passionate about the integration of biological and computational sciences in the healthcare industry. Interdisciplinary research experience in diverse fields (immunology, virology, structural biology and computational biology) and the ability to learn and adapt quickly.</p>
									<h4>Research Topics: Influenza virus; SARS-CoV-2; Antibody; Bioinformatics; Deep learning</h4>


									<hr class="major" />

									<h2>Skills</h2>
									<p><strong>Bioinformatics</strong> (NGS data analysis, Barcode/UMI/primer design, variants calling, statistical analysis, multiple sequence alignment; evolutionary analysis)</p>
									<p><strong>Programming Language</strong> (Python, R, Perl, Unix shell scripts, HTML; Familiar with Conda, Git, PyCharm)</p>
									<p><strong>Deep Learning</strong> (Familiar with Models: Transformer/CNNs/RNNs/GNNs; Frameworks: PyTorch/ TensorFlow)</p>
									<p><strong>Experimental Skills</strong> (Biochemical assay; DNA and RNA seq, high-throughput scanning; cell culture and mouse models)</p>
									<p><strong>Data Visualization & Design</strong> (Familiar with Cinema 4D, Adobe Premiere Pro, Photoshop)</p>

									<hr class="major" />

									<h2>Education & Training</h2>
									<dl>
										<dt><h4>2020-08 - Current</h4></dt>
										<dd>
											<p>Intern - gRED - Structural and Computational Biology, <i>Genentech, CA</i></p>
										</dd>
										<dt><h4>2020-08 - Current</h4></dt>
										<dd>
											<p>PhD student (Biochemistry/Bioinformatics), <i>University of Illinois Urbana - Champaign, IL</i></p>
										</dd>
										<dt><h4>2019-09 - 2020-08</h4></dt>
										<dd>
											<p>Research Assistant (Virology), <i>The University of Hong Kong - Hong Kong, China</i></p>
										</dd>
										<dt><h4>2015-09 - 2019-06</h4></dt>
										<dd>
											<p>Bachelor of Bioengineering (Bioengineering), <i>Chongqing University - Chongqing, China</i></p>
										</dd>
										<dt><h4>2018-07 - 2019-06</h4></dt>
										<dd>
											<p>Visiting Student (Systematic Biology), <i>Shenzhen Institute of Advanced Technology - Shenzhen, China</i></p>
										</dd>
									</dl>

									<hr class="major" />

									<h2>Publications</h2>
									<dl>
										<dt><strong>Preprints</strong></dt>
										<dd>
											<p>Liu T, <b>Wang Y</b>, Tan TJ, Wu NC, Brooke CB. The evolutionary potential of the influenza A virus hemagglutinin is highly constrained by intersegment epistasis. bioRxiv. <a href="https://doi.org/10.1101/2022.05.19.492711">DOI: https://doi.org/10.1101/2022.05.19.492711</a>.</p>
											<p>Yuan M, <b>Wang Y</b>, Lv H, Wilson IA, Wu NC. Molecular analysis of a public cross-neutralizing antibody response to SARS-CoV-2. bioRxiv. <a href="https://doi.org/10.1101/2022.05.17.492220">DOI: https://doi.org/10.1101/2022.05.17.492220</a>.</p>
											<p>Lei R, Tan TJC, Garcia AH, <b>Wang Y</b>, Teo C, Gopan G, Dargani ZT, Teo QW, Graham CS, Nair SK, Wu NC. Prevalence and mechanisms of evolutionary contingency in human influenza H3N2 neuraminidase. bioRxiv <a href="https://doi.org/10.1101/2022.02.24.481718">DOI: https://doi.org/10.1101/2022.02.24.481718</a>.</p>
											<p>Liang W, Tan TJC, <b>Wang Y</b>, Lv H, Sun Y, Bruzzone R, Mok CKP, Wu NC. Egg-adaptation pathway of human influenza H3N2 virus is contingent on natural evolution. bioRxiv <a href="https://doi.org/10.1101/2022.01.31.478576">DOI: https://doi.org/10.1101/2022.01.31.478576</a>.</p>
											<p>Lamers MM*, Breugem TI*, Mykytyn AZ*, <b>Wang Y</b>, Groen N, Knoops K, Schipper D, van der Vaart J, Koopman CD, Zhang J, Wu DC, van den Doel PB, Bestebroer T, GeurtsvanKessel CH, Peters PJ, Muraro MJ, Clevers H, Wu NC, Haagmans BL. Human organoid systems reveal in vitro correlates of fitness for SARS-CoV-2 B.1.1.7. bioRxiv <a href="https://doi.org/10.1101/2021.05.03.441080">DOI: https://doi.org/10.1101/2021.05.03.441080</a>.</p>
										</dd>
										<dt><strong>2022</strong></dt>
										<dd>
											<p><b>Wang Y*</b>, Yuan, M*, Lv, H, Peng, J, Wilson, IA, & Wu, NC. (2022). A large-scale systematic survey reveals recurring molecular features of public antibody responses to SARS-CoV-2. Immunity (2022). <a href="https://doi.org/10.1016/j.immuni.2022.03.019">DOI: https://doi.org/10.1016/j.immuni.2022.03.019</a>.</p>
										</dd>
										<dt><strong>2021</strong></dt>
										<dd>
											<p><b>Wang Y*</b>, Lei R*, Nourmohammad A, Wu NC. Antigenic evolution of human influenza H3N2 neuraminidase is constrained by charge balancing. eLife 10:e72516 (2021) <a href="https://doi.org/10.7554/eLife.72516">DOI: https://doi.org/10.7554/eLife.72516</a>.</p>
											<p>Lv H*, Tsang OTY*, So RTY, <b>Wang Y</b>, Yuan M, Liu H, Yip GK, Teo QW, Yihan Lin Y, Liang W, Wang J, Ng WW, Wilson IA, Peiris JSM, Wu NC#, Mok CKP#. Homologous and heterologous serological response to the N- terminal domain of SARS-CoV-2 in humans and mice. European Journal of Immunology 51:2296-2305 (2021) <a href="https://doi.org/10.1002/eji.202149234">DOI: https://doi.org/10.1002/eji.202149234</a>.</p>
											<p>Tan TJC*, Yuan M*, Kuzelka K, Padron GC, Beal JR, Chen X, <b>Wang Y</b>, Rivera-Cardona J, Zhu X, Stadtmueller BM, Brooke CB, Wilson IA#, Wu NC#. Sequence signatures of two public antibody clonotypes that bind SARS- CoV-2 receptor binding domain. Nature Communications 12:3815 (2021) <a href="https://doi.org/10.1038/s41467-021-24123-7">DOI: https://doi.org/10.1038/s41467-021-24123-7</a>.</p>
											<p>Lamers MM, Mykytyn AZ, Breugem TI, <b>Wang Y</b>, Wu DC, Riesebosch S, van den Doel PB, Schipper D, Bestebroer T, Wu NC, Haagmans BL. Human airway cells prevent SARS-CoV-2 multibasic cleavage site cell culture adaptation. eLife 10:e66815 (2021) <a href="https://doi.org/10.7554/eLife.66815">DOI: https://doi.org/10.7554/eLife.66815</a>.</p>
										</dd>
										<dt><strong>2020</strong></dt>
										<dd>
											<p>Lv H*, Wu NC*, Tsang OTY*, Yuan M, Perera RAPM, Leung WS, So RTY, Chan JMC, Yip GK, Chik TSH, <b>Wang Y</b>, Choi CYC, Lin Y, Ng WW, Zhao J, Poon LLM, Peiris JSM#, Wilson IA#, Mok CKP#. Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections. Cell Reports 31:107725 (2020) <a href="https://doi.org/10.1016/j.celrep.2020.107725">DOI: https://doi.org/10.1016/j.celrep.2020.107725</a>.</p>
										</dd>
										<dt><strong>2018</strong></dt>
										<dd>
											<p>Wang, Y*, Wei Zhou*, Feng Chen, Kaiyao Sun, Jixi Zhang, Ezgi Ã–zliselib, and Jessica M. Rosenholm. "Terbium complexes encapsulated in hierarchically organized hybrid MOF particles toward stable luminescence in aqueous media." CrystEngComm 20.30 (2018): 4225-4229.</p>
											<p>(*equal contributors)</p>
										</dd>
									</dl>
								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Search -->
								<section id="search" class="alt">
									<form id="form" role="search">
										<input type="text" name="query" id="query" placeholder="Search" aria-label="Search through site content"/>
									</form>
								</section>

							<!-- Menu -->
								<nav id="menu">
									<header class="major">
										<h2>Menu</h2>
									</header>
									<ul>
										<li><a href="index.html">Home</a></li>
										<li><a href="aboutme.html">About me</a></li>
										<li><a href="research.html">Research</a></li>
										<li><a href="blog.html">The blog</a></li>
										<li><a href="ideagenerator.html">Idea Generator</a></li>
										<li>
											<span class="opener">Bio-art</span>
											<ul>
												<li><a href="biopainting.html">Bio-painting</a></li>
												<li><a href="bioplus.html">Bio-plus</a></li>
												<li><a href="bioprocess.html">Bio-process</a></li>
												<li><a href="biovideo.html">Bio-video</a></li>
											</ul>
										</li>

									</ul>
								</nav>

							<!-- Section -->
								<section>
									<header class="major">
										<h2>Get in touch</h2>
									</header>
									<p>leave me a message</p>
									<ul class="contact">
										<li class="icon solid fa-envelope"><a href="mailto:yiquan2@illinois.edu">yiquan2@illinois.edu</a></li>
										<li class="icon solid fa-phone">(217) 721-4325</li>
									</ul>
								</section>

							<!-- Footer -->
								<footer id="footer">
									<p class="copyright">&copy; Yiquan Wang. All rights reserved. </p>
								</footer>

						</div>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>